example/lib/app_central.dart

```diff
@@ -1,54 +1,60 @@
 import 'package:flutter_chess_board/flutter_chess_board.dart';
 import 'package:universal_chess_driver/central.dart';
+import 'package:universal_chess_driver/peripherial.dart';
+
+class AppCentralRound implements CentralRound {
+  String? _variant;
+  String? _fen;
+  String? _lastMove;
+
+  AppCentralRound(this._variant, this._fen, this._lastMove);
+
+  String? get variant => _variant;
+  String? get fen => _fen;
+  String? get lastMove => _lastMove;
+}
 
 class AppCentral implements Central {
   ChessBoardController _chessController;
+  Peripherial? _peripherial = null;
```

Comment:
`Peripherial _peripherial = DummyPeripherial;`?

Reply:
I'm not sure. In Dart thanks to obj?.magic() we are having this behavior for free

Reply:
Right